src/core/Chip/StaticChip.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { StaticChip as OrigStaticChip, StaticChipProps } from './StaticChip';
+
+const StaticChip = (props: StaticChipProps) => {
```

Comment:
Having this at the top level does not feel necessary. Would be easier to read if all describes would just have their own definitions as this abstraction does not provide much additional value.

Reply:
You're absolutely right. Since all of the test cases have been given content I could simply remove this and did just that.